grails-app/domain/streama/SubtitlesResponse.groovy

```diff
@@ -0,0 +1,21 @@
+package streama
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+
+@JsonNaming(PropertyNamingStrategy.UpperCamelCaseStrategy.class)
+class SubtitlesResponse {
+
+  String subFileName
+
+  String subDownloadLink
+
+  String languageName
+
+  Long subDownloadsCnt
+
+  String subFormat
+
+  static constraints = {
+  }
```

Comment:
so in grails, everything in grails-app/domain is automatically mapped to the database. this is now true for `SubtitlesResponse` and `SubtitlesRequest` But really these seem more like helper classes to me. In that case, they should be moved to /src/main/groovy - or am I mistaken and these are intended for ORM?